src/nxt_cgroup.c

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (C) Andrew Clayton
+ * Copyright (C) F5, Inc.
+ */
+
+#include <nxt_main.h>
+#include <nxt_process.h>
+#include <nxt_malloc.h>
+#include <nxt_fs.h>
+#include <nxt_cgroup.h>
+
+#include <stdio.h>
+#include <string.h>
+#include <stdbool.h>
+#include <unistd.h>
+#include <limits.h>
```

Comment:
Please sort alphabetically, and libc headers before our own headers (when possible; I understand we need to include `<nxt_main.h>` as the first thing, and possibly other ugly stuff).

I'd like to conform to <https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes> as much as we can

Reply:
Yes, personally I put libc / library headers before project headers. So sounds good to me.